pkg/webhook/handlers/validation_test.go

```diff
@@ -39,7 +39,7 @@ func TestRegistryBackedValidator(t *testing.T) {
 	install.Install(scheme)
 	install.InstallValidations(registry)
 
-	log := klogr.New()
+	log := logf.Log
```

Comment:
This now uses global loggers rather than a test local logger.
Perhaps the original test author *wanted* to suppress (or capture) log messages in tests?

Reply:
not sure why this was done... removed that variable it was lonely only being used once